### DISCUSSION TITLE
logs/21w18

### DISCUSSION THREAD

**Discussion by laymonage** [laymonage/isle#7] [Comments]
Comments for https://laymonage.com/logs/21w18.

**Comment by laymonage** [comment]
Hello! :tada:

**Reply by carltongibson** [reply]
Hey @laymonage. Nice post. 

Good luck on the instrument! It’s finding a habit that’s key (I think) — can you make it so that’s it’s just something you do everyday, even if only for 10 mins? If so you win. 

Also, when are you open sourcing the repo for this GH Discussions integration, which is cool. 

😃

**Reply by laymonage** [reply]
Hey @carltongibson, thanks for stopping by!

>  It’s finding a habit that’s key (I think) — can you make it so that’s it’s just something you do everyday, even if only for 10 mins? If so you win.

I agree on that. It was hard to fit that 10 minutes when I had to go to uni early in the morning and go home in the evening, though! I admit that I didn't make much effort to try, but I also don't want it to be a chore. I'm quite confident I'll be able to do that now. I'll keep that in mind. Thanks! :smile:

> Also, when are you open sourcing the repo for this GH Discussions integration, which is cool.

When it's ready(-ish) :laughing:

I need to add documentation for the design decisions and current limitations. Even though it'll be in early alpha, I want people to be comfortable enough using it. (Plus, I'm a bit scared of people flooding the repo with issues that I'm already aware of. Need to make sure that people know what they're getting into before jumping in.)

Hopefully by the end of this week!

**Reply by carltongibson** [reply]
Nice 😊 "by the end of the week" is soon! 😀
